climada_petals/entity/exposures/openstreetmap/osm_dataloader.py

```diff
@@ -0,0 +1,945 @@
+        'oil' : {
+             'osm_keys' : ['pipeline','man_made','amenity','name'],
+             'osm_query' : """(pipeline='substation' and substance='oil') or
+                              (man_made='pipeline' and substance='oil') or
+                              man_made='petroleum_well' or 
+                              man_made='oil_refinery' or
+                              amenity='fuel'"""},
+
+        'power' : {
+              'osm_keys' : ['power','voltage','utility','name'],
+              'osm_query' : """power='line' or power='cable' or 
+                               power='minor_line' or power='plant' or
+                               power='generator' or power='substation' or
+                               power='transformer' or
+                               power='pole' or power='portal' or 
+                               power='tower' or power='terminal' or 
+                               power='switch' or power='catenary_mast' or
+                               utility='power'"""},
+        'wastewater' : {
+              'osm_keys' : ['reservoir_type','man_made','utility','natural',
+                            'name'],
+              'osm_query' : """reservoir_type='sewage' or
+                               (man_made='storage_tank' and content='sewage') or
+                               (man_made='pipeline' and substance='sewage') or
+                               substance='waterwaste' or 
+                               substance='wastewater' or
+                               (natural='water' and water='wastewater') or
+                               man_made='wastewater_plant' or
+                               man_made='wastewater_tank' or
+                               utility='sewerage'"""},
+         'food' : {
+             'osm_keys' : ['shop','name'],
+             'osm_query' : """shop='supermarket' or shop='greengrocer' or
+                              shop='grocery' or shop='general' or 
+                              shop='bakery'"""},
+         }
+
+
+class OSMRaw:
+    """
+    functions to obtain entire raw datasets from OSM, 
+    from different sources"""
+
+    def __init__(self):
+        self.geofabrik_url = 'https://download.geofabrik.de/'
+        self.planet_url = 'https://planet.openstreetmap.org/pbf/planet-latest.osm.pbf'
+
+    def _create_gf_download_url(self, iso3, file_format):
+        """
+        create string with download-api from geofabrik
+        
+        Parameters
+        ----------
+        iso3 : str
+            ISO3 code of country to download
+        file_format : str
+            Format in which file should be downloaded; ESRI Shapefiles ('shp') 
+            or osm-Protocolbuffer Binary Format ('pbf')
+        
+        Returns
+        -------
+        str : Geofabrik ownload-api for the requested country.
+        
+        See also
+        --------
+        DICT_GEOFABRIK for exceptions / special regions.
+        """
+
```

Comment:
```suggestion
         if iso3 not in DICT_GEOFABRIK.keys():
           LOGGER.error('The provided iso3 is not a recognised Geofabrik code')
```

Reply:
Good point. I added it in f0fcbd38, together with a warning for Russia, which is not RUS here.